kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -392,6 +392,40 @@ struct GetCategoryEarliestStale {
   }
 };
 
+struct GetStaleKeysSummary {
+  const bool read_only = true;
+  std::string description() const {
+    return "getStaleKeysSummary [BLOCK-VERSION-TO]\n"
+           "  return the number of stale keys per category type in the current blockchain\n"
+           "If BLOCK-VERSION-TO has been given, the method will summerize the stale keys from current genesis to "
```

Comment:
Nit: Could you please indent it?

Also, `summerize` is a typo. Maybe we can use `sum` instead.